apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -54,7 +54,9 @@ const DebugConsole = connect(
     removeWatchExpression,
     evalInCurrentScope: actions.evalInCurrentScope,
     appendLog: actions.appendLog,
-  }
+  },
+  null,
+  {withRef: true}
```

Comment:
The `withRef` option is [documented to be available on `connectAdvanced`](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectadvancedselectorfactory-connectoptions) but [not on `connect`](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options).  Is this an error in their documentation, or are we using an outdated Redux?  Also, I don't see a call to `getWrappedInstance` anywhere, so what does this get us?